templates/cmake/cmake/psl1ght.cmake

```diff
@@ -0,0 +1,73 @@
+if(NOT $ENV{PSL1GHT} STREQUAL "")
+    set(PSL1GHT $ENV{PSL1GHT})
+endif()
+
+add_custom_target(PS3Package)
+
+macro(setupPSL1GHT target)
+    if(NOT PSL1GHT)
+        message(FATAL_ERROR "Location of PSL1GHT SDK not set")
+    endif()
+    set(CMAKE_C_COMPILER ${PSL1GHT}/ppu/bin/powerpc64-ps3-elf-gcc)
+    set(CMAKE_CXX_COMPILER ${PSL1GHT}/ppu/bin/powerpc64-ps3-elf-g++)
```

Comment:
Add these after line 12:
```cmake
set(CMAKE_CXX_LINKER ${PSL1GHT}/ppu/bin/powerpc64-ps3-elf-gcc)
set(CMAKE_C_LINKER ${PSL1GHT}/ppu/bin/powerpc64-ps3-elf-gcc)
```

I think setting prefix as a variable at the top of the file cleans up your code a bit.